__tests__/app.test.js

```diff
@@ -99,4 +98,27 @@ describe("GET /api/articles/:article_id", () => {
             expect(body.msg).toBe("Article not found")
         })
       });
+})
+describe("GET /api/articles", () => {
+  test("200 - responds with array of article objects that have correct keys sorted by date descending", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("created_at", { descending: true })
+        expect(Array.isArray(articles)).toBe(true);
```

Comment:
should check length of array if asserting inside a forEach as it would auto pass by default